index.html

```diff
@@ -168,6 +169,39 @@ <h3>## Hamburg</h3>
       </div>
     </article>
 
+    <article id="offers">
+      <div class="separator">
+        <img src="assets/sticker/07.png" />
+        <img src="assets/sticker/08.png" />
+        <img src="assets/sticker/09.png" />
+        <img src="assets/sticker/10.png" />
+        <img src="assets/sticker/11.png" />
+        <img src="assets/sticker/12.png" />
+      </div>
+
+      <h2># Offers</h2>
+
+      <div>
+        <h3>## Food</h3>
+
+        <p>
+          This year we only want to provide vegetarian food like last year. In the morning there will be breakfast with sandwiches.
```

Comment:
(sorry for the late reaction)
the food is actually vegan, not just vegetarian!

Reply:
Done :)